File: solutions/uri/1019/1019.py
import sys

h = 0
m = 0

for t in sys.stdin:
    t = int(t)

    if t >= 60 * 60:
        h = t // (60 * 60)
        t %= 60 * 60

    if t >= 60:
        m = t // 60
        t %= 60

    print(f"{h}:{m}:{t}")

    h = 0
    m = 0

File: solutions/beecrowd/1019/1019.js
const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  terminal: false
});

(() => {
  rl.on('line', (line) => {
    let n = parseInt(line)
    let h = 0
    let m = 0

    if (n >= 60 * 60) {
      h = Math.floor(n / (60 * 60))
      n %= 60 * 60
    }
    if (n >= 60) {
      m = Math.floor(n / 60)
      n %= 60
    }

    console.log(`${h}:${m}:${n}`)
  })
})()
